microsoft-edge/webview2/reference/javascript/index.yml

```diff
@@ -4,7 +4,7 @@ metadata:
   title: JavaScript APIs in WebView2
 name: WebView2Script
 type: package
-summary: 'Web pages can use these JavaScript APIs to communicate with WebView2 objects.' # TODO: Add WebView2 JavaScript API overview. See ADO 41704272.
+summary: 'The Microsoft Edge WebView2 control lets you embed web content into native applications. You can use the following `window.chrome.webview` JavaScript APIs along with native WebView2 APIs to communicate between native code and web code. '
```

Comment:
Remove the trailing space before final single quote.

corp style guide gives some interesting wording I've been trying, very direct:

'Use the Microsoft Edge WebView2 control to embed web content into native applications. You can use the following `window.chrome.webview` JavaScript APIs along with native WebView2 APIs to communicate between native code and web code. '

I'm not able to enter a Suggestion (maybe restart browser or F5 will enable).

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/let-lets - "**Don't use lets** (or allows or enables) to refer to things that Microsoft or Microsoft apps make possible for the customer. Instead, focus on what the customer wants to **do**.

Example
Present schedule information in a variety of ways using Microsoft Project."